managed/ui/src/components/universes/UniverseForm/ClusterFields.js

```diff
@@ -492,16 +493,24 @@ export default class ClusterFields extends Component {
       this.props.fetchUniverseResources(nextProps.universe.universeConfigTemplate.data);
     }
     // If nodeInstanceList changes, fetch number of available nodes
-    if (
-      getPromiseState(nodeInstanceList).isSuccess() &&
-      getPromiseState(this.props.cloud.nodeInstanceList).isLoading()
```

Comment:
is this condition removed intentionally?

Reply:
Yeah. Actually, I think we don't need this `isLoading()` here in `willRecieveProps`. This condition will get fail if you come back to the `primary` cluster from `read replica`.  And now we're setting up `maxNodes` based on region selection so I believe it's fine. 

Please lemme know if this needs to be corrected.